### PR TITLE
QA: Add Bein Sports 1 & 2 English and try to combat Al Jazeera messiness

### PR DESCRIPTION
**Pull request by Dum4G**

### CODE REVIEW

File: channels/qa.m3u [hunk 1]
```diff
@@ -19,42 +19,48 @@ https://live-hls-web-ajd.getaj.net/AJD/index.m3u8
...
+#EXTINF:-1 tvg-id="AlJazeeraEnglish.qa" tvg-name="Al Jazeera English" tvg-country="AU" tvg-language="English" tvg-logo="http://i.mjh.nz/.images/tv.46.png" group-title="News",Al Jazeera English (1080p) [Geo-blocked]
 https://i.mjh.nz/au/Sydney/tv.46.m3u8
 #EXTINF:-1 tvg-id="AlJazeeraEnglish.qa" tvg-name="Al Jazeera English" tvg-country="INT" tvg-language="English" tvg-logo="http://www.aljazeera.com/assets/images/aj-logo-lg-124.png" group-title="News",Al Jazeera English
 https://live-hls-web-aje.getaj.net/AJE/01.m3u8
 #EXTINF:-1 tvg-id="AlJazeeraEnglish.qa" tvg-name="Al Jazeera English" tvg-country="INT" tvg-language="English" tvg-logo="https://i.imgur.com/BB93NQP.png" group-title="News",Al Jazeera English (1080p)
 https://live-hls-web-aje.getaj.net/AJE/index.m3u8
-#EXTINF:-1 tvg-id="ALJazeeraMubashe.qa" tvg-name="AL Jazeera Mubashe" tvg-country="QA" tvg-language="" tvg-logo="" group-title="",AL Jazeera Mubashe
+#EXTINF:-1 tvg-id="Al Jazeera Mubasher.qa" tvg-name="Al Jazeera Mubasher" tvg-country="ARAB" tvg-language="Arabic" tvg-logo="https://i.imgur.com/xx6uOhZ.png" group-title="Legislative",Al Jazeera Mubasher
```
Comment (LaneSh4d0w) [inline channels/qa.m3u:28]: Nice try, but the tvg-ids do not contain spaces.

**Review by BellezaEmporium — CHANGES_REQUESTED** [review]

File: channels/qa.m3u (hunk 1 above)
Comment (Dum4G) [reply channels/qa.m3u]: My bad

**Review by BellezaEmporium — APPROVED** [review]

File: channels/qa.m3u [hunk 2]
```diff
@@ -19,42 +19,48 @@ https://live-hls-web-ajd.getaj.net/AJD/index.m3u8
...
 http://live-hls-web-ajm.getaj.net/AJM/01.m3u8
-#EXTINF:-1 tvg-id="beINSports1Asia.qa" tvg-name="beIN Sports 1 Asia" tvg-country="QA" tvg-language="" tvg-logo="https://static.epg.best/qa/beINSports1.qa.png" group-title="",beIN Sports 1 Asia
+#EXTINF:-1 tvg-id="beINSports1Asia.qa" tvg-name="beIN Sports 1 Asia" tvg-country="QA" tvg-language="" tvg-logo="https://static.epg.best/qa/beINSports1.qa.png" group-title="Sports",beIN Sports 1 Asia
 https://liveanevia.mncnow.id/live/eds/BeIN1/sa_dash_vmx/BeIN1.mpd
-#EXTINF:-1 tvg-id="DohatAlQuraan.qa" tvg-name="Dohat Al-Quraan" tvg-country="QA" tvg-language="" tvg-logo="" group-title="",Dohat Al-Quraan
-http://45.92.69.131:8001/play/a01d
+#EXTINF:-1 tvg-id="beINSports1English.qa" tvg-name="beIN Sports 1 English" tvg-country="" tvg-language="English" tvg-logo="https://www.lyngsat.com/logo/tv/bb/bein-sports-english-1-qa.png" group-title="Sports",beIN Sports 1 English
+#EXTVLCOPT:http-user-agent=Mozilla/5.0 Macintosh; Intel Mac OS X 10_14_5 AppleWebKit/537.36 KHTML, like Gecko Chrome/76.0.3809.25 Safari/537.36
+https://www.livedoomovie.com/02_epl1_720p_UK/chunklist.m3u8
```
Comment (adamo331) [inline channels/qa.m3u:60]: tested in vlc not working

File: channels/qa.m3u [hunk 3]
```diff
@@ -19,42 +19,48 @@ https://live-hls-web-ajd.getaj.net/AJD/index.m3u8
...
+#EXTINF:-1 tvg-id="beINSports2English.qa" tvg-name="beIN Sports 2 English" tvg-country="" tvg-language="English" tvg-logo="https://www.lyngsat.com/logo/tv/bb/bein-sports-english-2-qa.png" group-title="Sports",beIN Sports 2 English
+#EXTVLCOPT:http-user-agent=Mozilla/5.0 Macintosh; Intel Mac OS X 10_14_5 AppleWebKit/537.36 KHTML, like Gecko Chrome/76.0.3809.25 Safari/537.36
+https://www.livedoomovie.com/02_epl2_720p_UK/chunklist.m3u8
```
Comment (adamo331) [inline channels/qa.m3u:63]: tested in vlc not working

File: channels/qa.m3u (hunk 2 above)
Comment (LaneSh4d0w) [reply channels/qa.m3u:60]: Both worked on the time I've tested them, apparently the streams this host has got aren't stable.

Comment (Dum4G) [reply channels/qa.m3u:60]: image
test properly. user agent is used for a reason

**Pull request merged**